facade/src/main/java/com/redhat/lightblue/migrator/features/TogglzRandomUserProvider.java

```diff
@@ -0,0 +1,21 @@
+package com.redhat.lightblue.migrator.features;
+
+import org.togglz.core.user.FeatureUser;
+import org.togglz.core.user.SimpleFeatureUser;
+import org.togglz.core.user.UserProvider;
+
+/**
+ * Call {@link TogglzRandomUsername#init()} to generate new random username before
+ * this provider is used. {@link DAOFacadeBase} apis already handle that.
+ *
+ * @author mpatercz
+ *
+ */
+public class TogglzRandomUserProvider implements UserProvider {
```

Comment:
No. It's for creating usernames which remain the same for the duration of the api call. I'm abusing Togglz here, it's not very suitable for request based gradual rollout.